Extra test data for Nonpareil

fgvieira opened the pull request:
Adding extra test data to nonpareil.

vladsavelyev commented:
I totally missed all the PR in test-data, sorry 🤦 

How are these examples different from the existing `sample.json`?

fgvieira commented:
Before there was a single file with two samples with model.
I added an example with a single sample (`X.json`), and switched one sample in the other file (`samples.json`) with one with no model (because there as a bug that would give an error with samples with no model).

But feel free to change the names.
